runHarness/Parameters.pm

```diff
@@ -2352,7 +2352,7 @@ $parameters{"nginxUseNamedVolumes"} = {
 
 $parameters{"nginxCacheStorageClass"} = {
 	"type"      => "=s",
-	"default"   => "fast",
+	"default"   => "weathervaneSC",
```

Comment:
Should the weathervane.config* files be similarly updated?

Reply:
Yes.  Done.